src/ansys/mapdl/core/errors.py

```diff
@@ -306,26 +307,73 @@ def wrapper(*args, **kwargs):
                 old_handler = signal.signal(signal.SIGINT, handler)
 
         # Capture gRPC exceptions
-        try:
-            out = func(*args, **kwargs)
-        except grpc.RpcError as error:
-            # Custom errors
-            if error.code() == grpc.StatusCode.RESOURCE_EXHAUSTED:
-                if "Received message larger than max" in error.details():
-                    try:
-                        lim_ = int(error.details().split("(")[1].split("vs")[0])
-                    except IndexError:
-                        lim_ = int(512 * 1024**2)
-
-                    raise MapdlgRPCError(
-                        f"RESOURCE_EXHAUSTED: {error.details()}. "
-                        "You can try to increase the gRPC message length size using 'PYMAPDL_MAX_MESSAGE_LENGTH'"
-                        " environment variable. For instance:\n\n"
-                        f"$ export PYMAPDL_MAX_MESSAGE_LENGTH={lim_}"
+        n_attempts = 3
+        initial_backoff = 0.05
+        multiplier_backoff = 3
+
+        i_attemps = 0
+
+        while True:
+            try:
+                out = func(*args, **kwargs)
+
```

Comment:
As long as you are confident that your server will handle well retrying a request after failure then sure, this looks like a good approach.

Reply:
Pressumably, the call failed and didn't reach the server. I am 95% sure, but i wanted additional feedback in case I'm missing something. Thank you @RobPasMue for your feedback!